packages/apps/graph/src/graph/index.ts

```diff
@@ -6,14 +6,19 @@ import './objects/fungible-chain-account';
 import './objects/gas-limit-estimation';
 import './objects/graph-configuration';
 import './objects/guard';
+import './objects/meta';
 import './objects/miner-key';
 import './objects/non-fungible-account';
 import './objects/non-fungible-chain-account';
+import './objects/payload';
 import './objects/signer';
 import './objects/token';
 import './objects/token-info';
 import './objects/total-count';
-import './objects/transaction';
+// import './objects/transaction';
+import './objects/cmd';
+import './objects/transaction-result';
+import './objects/transactions';
```

Comment:
I think we want to keep it named `transaction`, since everything else is singular as well